docs/user-guide/basic-usage.md

```diff
@@ -39,10 +39,58 @@ spec:
           serviceName: myserviceb
           servicePort: 80
 ```
-
+ 
 When you apply this yaml, 2 ingress resources will be created managed by the **ingress-nginx** instance. Nginx is configured to automatically discover all ingress with the `kubernetes.io/ingress.class: "nginx"` annotation.
 Please note that the ingress resource should be placed inside the same namespace of the backend resource.
 
+If cluster version >= 1.19 the Ingress resource above will not work.
```

Comment:
Should you directly write here which versions are not available?

Reply:
Sorry not sure what you mean,
New ingress was introduced with Kubernetes version 1.19, as stated here https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#ingress-graduates-to-general-availability

it is available for all version >=1.19 at current time

Reply:
I mean the Ingress's API version.

Reply:
All versions except the one I added, are no longer valid